debian/install

```diff
@@ -1,2 +1,18 @@
 debian/dosemu.desktop usr/share/applications/
 etc/dosemu2.alias etc/X11/fonts/misc/
+usr/bin/dosdebug
```

Comment:
Why?
This install is only in addition to
"make install".
Or is it needed for multi-package
build?

Reply:
Yeah that seemed to be needed, but I suspect it's also because I'm not fully understanding the multi-package build yet. Out of scope for this, but I was also thinking that `dosdebug` could be put in its own package.

Reply:
Yes, I also do remember that multipackage
build was more difficult. You probably did
things right.
But I am not sure we need this in.
Building contrib packages from the main
source will create the illusion that we still
depend on the non-free stuff.
Also a python dep, and msexpand build...
Not sure why do I need all that.
Why not to make this separately?

Reply:
> Building contrib packages from the main
> source will create the illusion that we still
> depend on the non-free stuff.

I feared the same. I tried to explain this with the description.


>  Why not to make this separately?

Should be fine yes. Then for now, treat the package separation in this PR as my proposal on what to move out to separate repos. (Note that this still misses dosemu-msexpand for example.)

Reply:
Ideally we can just package
msexpand and make it conflict
with mscompress package.
Then ask debian to take it.

Reply:
If all features in the version that Debian has are also in this one that should work. (I didn't test with formats different/newer than the MS-DOS 6.22 one.)